generators/client/templates/angular/.eslintignore.ejs

```diff
@@ -7,3 +7,4 @@ target/
 build/
 node/
 postcss.config.js
+dist/
```

Comment:
Can we leverage existing build locations like `target` with maven and `build` with Gradle.

Reply:
Yes, that's good idea! I'll close this PR and update guide in jhipster/ng-jhipster#106 suggesting to use `target/dist` if using **Maven** and `build/dist` is using **Gradle** or **skipping server**.